cg/store/models.py

```diff
@@ -991,9 +991,12 @@ class Order(Base):
     customer: Mapped[Customer] = orm.relationship(foreign_keys=[customer_id])
     order_date: Mapped[datetime] = mapped_column(default=datetime.now)
     ticket_id: Mapped[int] = mapped_column(unique=True, index=True)
-    workflow: Mapped[str] = mapped_column(types.Enum(*(workflow.value for workflow in Workflow)))
     is_open: Mapped[bool] = mapped_column(default=True)
 
+    @property
+    def workflow(self) -> Workflow:
+        return self.cases[0].data_analysis
```

Comment:
Potential issue with regards to MAF cases: For each fastq order we will have only one of its cases (the "real" one) being of the fastq workflow, while we have a number of MAF cases under MIP-DNA. This implementation would make the fastq orders have a workflow which quite often says MIP-DNA

Reply:
I think a new case is created attached to a different order for this ( I will check). So ideally these things should not mingle.

Reply:
I just checked. Indeed we put the MAF case onto the original order instead of making a separate one...

Reply:
Yeah. I still don't quite understand MAF cases but I think it makes sense to have them in the same order so you know what cases they are tied to?

Reply:
I would be very happy if they could be in another order so yeah?